zcash_script/src/script/mod.rs

```diff
@@ -417,6 +439,30 @@ impl Evaluable for Code {
     }
 }
 
+impl Asm for Code {
+    fn to_asm(&self, attempt_sighash_decode: bool) -> String {
+        let mut v = Vec::new();
+        let is_unspendable = self.is_unspendable();
+        for r in self.parse_strict(&interpreter::Flags::StrictEnc) {
+            match r {
+                Ok(op) => v.push(op.to_asm(attempt_sighash_decode && !is_unspendable)),
+                Err(_) => {
+                    v.push("[error]".to_string());
+                    break;
+                }
+            }
+        }
+        v.join(" ")
+    }
+}
+
+/// Implement Display for any type that implements Asm
```

Comment:
This comment looks like it was for a generic impl that doesn’t exist any more.

Reply:
Oops. Thanks for catching it. Deleted the comment